tests/Unit/Routes/WebRoutesTest.php

```diff
@@ -1,20 +1,153 @@
 <?php
 
-namespace Tests\Unit\Routes;
-
 use Tests\TestCase;
```

Comment:
Did you remove namespace intentionally?

Reply:
Yes. PHPStorm was winging that App\Centre and App\User weren't in PHP's class auto-search scope if the namepace was there. Not sure i beileve it...